res/css/views/rooms/_MessageComposer.scss

```diff
@@ -37,6 +37,7 @@ limitations under the License.
     }
 
     .mx_MessageComposer_row {
+        overflow-x: clip;
```

Comment:
We can't use clip without the `hidden` fallback due to lack of Safari support

![image](https://user-images.githubusercontent.com/2403652/173337173-dcdde31a-4b2f-4e8d-84aa-7e6b333e8701.png)

Reply:
It’s always safari ugh :(